p4-16/spec/P4-16-spec.mdk

```diff
@@ -4700,9 +4701,10 @@ optInitializer
     ;
 ~ End P4Grammar
 
-Variable declarations without an initializer are uninitialized (except for header
-stacks, which have their `nextIndex` counter initialized to `0`,
-as discussed in [#sec-expr-hs]). The language places few restrictions on
+Variable declarations without an initializer are uninitialized (except for
+headers and other header-related types, which are initialized to invalid in the
+same way as described for direction `out` parameters in Section
+[#sec-calling-convention]). The language places few restrictions on
```

Comment:
Rather than repeating all of the text used to describe how direction `out` parameters must be initialized, I propose referring to that new text from here.